src/__tests__/steps/index.test.ts

```diff
@@ -164,6 +170,6 @@ describe('service entities/relationships', () => {
         }),
       ]),
     );
-    expect(serviceUserRelationships).toHaveLength(150);
+    expect(serviceUserRelationships).toHaveLength(45);
```

Comment:
Is the change here because of different data or steps being disabled?

Reply:
Just due to different data.